chrome-developer-tools/chrome-developer-tools.md

```diff
@@ -27,22 +27,24 @@ Chrome Developer Tools help tremendously with front-end development (HTML, CSS &
 - How to access the JavaScript console to run and debug code
 - Participants will learn the various keyboard shortcuts to open Chrome Developer Tools
 
-### Supplemental Materials
+### Materials
+- [Chrome Developer Tools (video walkthrough of slides)](https://drive.google.com/file/d/1LV6lOI0MRKUVSaDlSipiy6JX8EtEaYjH/view?usp=sharing)
+- [Chrome Developer Tools (slides)](https://docs.google.com/presentation/d/1z8aIzOxV5L-zW3MZmZhX67DgyLq_4ErO4yTS4CIp7R0/edit?usp=sharing)
 
+### Supplemental Materials
 - [Quick Source Viewer Chrome Extension](https://chrome.google.com/webstore/detail/quick-source-viewer/cfmcghennfbpmhemnnfjhkdmnbidpanb?hl=en-US)
 
 ### Lesson
 - [Chrome Developer Tools (video walkthrough of slides)](https://drive.google.com/file/d/1LV6lOI0MRKUVSaDlSipiy6JX8EtEaYjH/view?usp=sharing)
-
 - [Chrome Developer Tools (slides)](https://docs.google.com/presentation/d/1z8aIzOxV5L-zW3MZmZhX67DgyLq_4ErO4yTS4CIp7R0/edit?usp=sharing)
 
```

Comment:
The two things that are in the **Lesson** section are already mentioned in the **Material** section. So I think Lesson section could be removed.

Reply:
@alodahl @Kundan28 
what about naming material section as Lesson also?
i.e. material / lesson

Reply:
No, I don't think it would look good.

Reply:
Done, I made the request changes expect the slideshow guidelines once.

Reply:
I can't review the Slideshow as I don't have edit access. But rest of the thing is good. And this PR is ready to be merged.

Reply:
But the slideshow looks good to me.